src/plugins/ds_redis.c

```diff
@@ -2641,12 +2641,13 @@ srpds_load_and_copy_data(redisContext *ctx, const char *mod_ns_src, const char *
  * @param[in] ctx Libyang context.
  * @param[in] xpaths Array of XPaths.
  * @param[in] xpath_cnt XPath count.
+ * @param[in] discard_items If sysrepo discard-items nodes should be retrieved as well.
```

Comment:
Please just add `(always for operational DS)`. I was a bit confused initially.

Reply:
Changed it so that `srpds_process_load_paths()` functions in both mongo and redis DS plugins are similar.